test/integration/helpers/semantic-with-svg-utils.ts

```diff
@@ -28,8 +28,8 @@ export function expectEndEventBpmnElement(bpmnElement: BpmnElement, expected: Ex
   expectSvgEvent(bpmnElement.htmlElement);
 }
 
-export function expectSequenceFlowBpmnElement(bpmnElement: BpmnElement, expected: ExpectedBaseBpmnElement): void {
-  expectSequenceFlow(bpmnElement.bpmnSemantic, expected);
+export function expectSequenceFlowBpmnElement(bpmnElement: BpmnElement, expected: ExpectedFlowElement): void {
```

Comment:
**suggestion (non-blocking):** To avoid to cast `bpmnElement` in the next line, we can change its type in the method signature.

```suggestion
export function expectSequenceFlowBpmnElement(bpmnElement: EdgeBpmnSemantic, expected: ExpectedFlowElement): void {
```

Reply:
Unfortunately, this is not possible IMHO. Here we need to access both the `bpmnSemantic` and `htmlElement` properties of `BpmnElement`.
So I don't see how I can change the signature of the function.